src/DIRAC/WorkloadManagementSystem/Utilities/test/Test_JobModel.py

```diff
@@ -0,0 +1,218 @@
+"""Test the JobModel class and its validators."""
+
+# pylint: disable=invalid-name
+
+from unittest.mock import patch
+import pytest
+from pydantic import ValidationError
+
+from DIRAC import S_OK
+from DIRAC.WorkloadManagementSystem.Utilities.JobModel import BaseJobDescriptionModel, JobDescriptionModel
+
+EXECUTABLE = "dirac-jobexec"
+SETUP = "diracSetup"
+OWNER = "owner"
+ONWERDN = "ownerDN"
+OWNERGROUP = "ownerGroup"
+VO = "vo"
+
+
+@pytest.mark.parametrize(
+    "cpuTime",
+    [
+        100,  # Lower bound
+        86400,  # Default
+        500000,  # Higher bound
+    ],
+)
+def test_cpuTimeValidator_valid(cpuTime: int):
+    """Test the cpuTime validator."""
+    BaseJobDescriptionModel(executable=EXECUTABLE, cpuTime=cpuTime)
+
+
+@pytest.mark.parametrize(
+    "cpuTime",
+    [
+        1,  # Too low
+        100000000,  # Too high
+    ],
```

Comment:
Would the following tests make sense?
- `cpuTime` = "azerty"
- `cpuTime` = None

Same comment for the other tests?